### PR TITLE
feat: Add accept header in resolver http binding

### LINKED ISSUE #1384: Add accept header in resolver http binding
Add accept "application/did+ld+json" header

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1384

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/vdri/httpbinding/resolver.go [hunk 1]
```diff
@@ -26,14 +30,21 @@ type didResolution struct {
...
+
+	resp, err := v.client.Do(req)
 	if err != nil {
 		return nil, fmt.Errorf("HTTP Get request failed: %w", err)
 	}
 
 	defer closeResponseBody(resp.Body)
 
-	if resp.StatusCode == http.StatusOK {
+	if resp.StatusCode == http.StatusOK && resp.Header.Get("Content-type") == "application/did+ld+json" {
```
Comment (sandrask) [inline pkg/vdri/httpbinding/resolver.go]: use above mentioned constant for comparison

Comment (rolsonquadras) [inline pkg/vdri/httpbinding/resolver.go]: can we use the constant defined in line 21 here ? ie `didLDJson = "application/did+ld+json"`.

Comment (fqutishat) [reply pkg/vdri/httpbinding/resolver.go]: done

File: pkg/vdri/httpbinding/resolver.go [hunk 2]
```diff
@@ -26,14 +30,21 @@ type didResolution struct {
...
+
+	resp, err := v.client.Do(req)
 	if err != nil {
 		return nil, fmt.Errorf("HTTP Get request failed: %w", err)
 	}
 
 	defer closeResponseBody(resp.Body)
 
-	if resp.StatusCode == http.StatusOK {
+	if resp.StatusCode == http.StatusOK && resp.Header.Get("Content-type") == didLDJson {
```
Comment (sandrask) [inline pkg/vdri/httpbinding/resolver.go]: I will create a ticket to add reading response body in case of error too

Comment (fqutishat) [reply pkg/vdri/httpbinding/resolver.go]: done

File: pkg/vdri/httpbinding/resolver.go [hunk 3]
```diff
@@ -17,6 +17,10 @@ import (
 	vdriapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdri"
 )
 
+const (
+	didLDJson = "application/did+ld+json"
+)
```
Comment (soluchok) [inline pkg/vdri/httpbinding/resolver.go]: nit: should be in one line

File: pkg/vdri/httpbinding/resolver.go [hunk 4]
```diff
@@ -26,28 +30,35 @@ type didResolution struct {
...
+	resp, err := v.client.Do(req)
 	if err != nil {
 		return nil, fmt.Errorf("HTTP Get request failed: %w", err)
 	}
 
 	defer closeResponseBody(resp.Body)
 
-	if resp.StatusCode == http.StatusOK {
-		var gotBody []byte
+	var gotBody []byte
```
Comment (soluchok) [inline pkg/vdri/httpbinding/resolver.go]: Redundant definition. Lint should not complain about `gotBody, err :=`

**Pull request merged**